controllers/news.js

```diff
@@ -0,0 +1,31 @@
+const createHttpError = require('http-errors')
+const { endpointResponse } = require('../helpers/success')
+const { getNewById } = require('../services/news')
+
+const getNews = async (req, res, next) => {
```

Comment:
```suggestion
const listNews = async (req, res, next) => {
```
ayer intentamos normalizar el nombre de las funciones para que todos usemos los mismos, quedamos en usar list en vez de get

Reply:
+1

Reply:
Modificado